admin/views/pupils-not-taking-the-check/select-pupils.ejs

```diff
@@ -1,97 +1,72 @@
-<%- partial('partials/_breadcrumbs_and_sign_out', { breadcrumbs }) %>
-
-<div class="grid-row" id="pupils-not-taking-checks" aria-describedby="heading-xlarge">
-
-    <div class="column-two-thirds">
-        <% if (messages.info && messages.info.length > 0) { %>
-        <div class="info-message">
-            <%= messages.info.join('<br/>') %>
-        </div>
-        <% } %>
-
-        <%- partial('partials/_readonly', {availabilityData: {hdfSubmitted}}) %>
-
-        <h1 class="heading-xlarge" id="heading-xlarge">Pupils not taking the check</h1>
-
-        <%- partial('partials/_session_expiry') %>
-
-        <p class="lede">
-            All pupils must be considered for the multiplication tables check at the end of year 4.
-            If a pupil is not going to take the check, a reason must be provided.
-        </p>
-
-        <div class="button-wrapper">
-            <% if (hdfSubmitted) { %>
-            <a class="button-secondary" disabled>Select pupil and reason</a>
-            <% } else { %>
-            <a class="button" href="/pupils-not-taking-the-check/select-pupils" >Select pupil and reason</a>
+<%- partial('partials/_gds_breadcrumbs_and_sign_out', { breadcrumbs }) %>
+<main class="govuk-main-wrapper" id="main-content" role="main">
+    <div class="govuk-grid-row" id="pupils-not-taking-checks">
+        <div class="govuk-grid-column-two-thirds">
+            <% if (messages.info && messages.info.length > 0) { %>
+                <div class="govuk-info-message govuk-!-margin-bottom-7">
+                    <%= messages.info.join('<br/>') %>
+                </div>
             <% } %>
-        </div>
-
+            <%- partial('partials/_readonly', { availabilityData: { hdfSubmitted } }) %>
+            <h1 class="govuk-heading-xl" aria-describedby="heading-xlarge">Pupils not taking the check</h1>
+            <%- partial('partials/_session_expiry') %>
+            <p class="govuk-body">
+                All pupils must be considered for the multiplication tables check at the end of year 4.
+                If a pupil is not going to take the check, a reason must be provided.
+            </p>
+        <% if (hdfSubmitted) { %>
+            <button type="submit" disabled="disabled" aria-disabled="true" class="govuk-button govuk-button--disabled" data-module="govuk-button">
+                Select pupil and reason
+            </button>
+        <% } else { %>
+            <button type="button" onclick="window.location.replace('/pupils-not-taking-the-check/select-pupils')" class="govuk-button" data-module="govuk-button">
```

Comment:
I think if we want to send the user to another location then we should use a href styled to look like a button.

Reply:
@jon-shipley faking a link as a button is against both GDS and accessibility rules

Reply:
@jon-shipley updated based on discussions on all buttons with onclick handlers